_posts/2024-07-12-ecrs-summary.markdown

```diff
@@ -0,0 +1,42 @@
+
+I attended ECRS to fill in the coral-loving world on my work [*predicting where coral will survive best in futures experiencing the environmental conditions caused by anthropogenic climate change*](https://orlando-code.github.io/research.html). The objective here is to guide our precious reef conservation efforts to those areas where corals are most likely to survive under climate change. This requires robust predictions of environmental suitability at spatial scales relevant to these conservation projects.
+
+While it’s quite the challenge, initial results look promising. In short, a fairly simple machine learning model is able to nicely recreate the distribution of reefs that it would expect to see, given only information about depth, and how warm, how salty, how fast-moving, how sunny (et cetera et cetera) the oceans off Australia have been over the last ~150 years. 
+
+<figure style="width: 100%" class="align-center">
+  <img src="/assets/images/posts/ecrs_summary/GBR_predicted.png" alt="Prediction of reef distribtuion by machine learning model (similar to observed)." style="width:49%">
+  <img src="/assets/images/posts/ecrs_summary/GBR_true.png" alt="Observed present-day reef distribution (similar to predicted)." style="width:49%">
+  <figcaption>
+  <strong> Model prediction of Great Barrier Reef.</strong> Left: Model prediction of coral density based on historic environmental timeseries. Right: Observed geographical distribution of the present-day reef.
+  </figcaption>
+</figure>
+
+<figure style="width: 100%" class="align-center">
+  <img src="/assets/images/posts/ecrs_summary/zGBR_predicted.png" alt="Prediction of reef distribtuion by machine learning model (similar to observed)." style="width:49%">
+  <img src="/assets/images/posts/ecrs_summary/zGBR_true.png" alt="Observed present-day reef distribution (similar to predicted)." style="width:49%">
+  <figcaption>
+  <strong> Zooming in on the model prediction of Great Barrier Reef.</strong> Left: Model shies away from predicting the highest-density coral areas, and edges of coral patches. Right: Observed geographical distribution of the present-day reef.
+  </figcaption>
+</figure>
+
+How do we know this information? Climate models! The beauty of being able to learn the conditions favoured by corals from these is that these models also contain our best guess as to what these conditions will look like far into the future as a result of various different roadmaps of greenhouse gas emissions. The ability to predict present-day coral coverage from this data therefore indicates that we can predict (under certain assumptions) where conditions will be favourable in the coming century. Longtermist conservation projects here we come! 
```

Comment:
not clear what 'this information' refers to. Perhaps 'how do we get all this information about environmental conditions?'